arenadata/Dockerfile.ubuntu

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:22.04 AS base
+
+COPY README.Ubuntu.bash ./
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN set -eux; \
+    ./README.Ubuntu.bash; \
+    rm README.Ubuntu.bash; \
+# The en_US.UTF-8 locale is needed to run GPDB
+    locale-gen en_US.UTF-8; \
+# To run sshd directly, but not using `/etc/init.d/ssh start`
+    mkdir /run/sshd; \
+# Alter precedence in favor of IPv4 during resolving
+    echo 'precedence ::ffff:0:0/96  100' >> /etc/gai.conf; \
+# Packages for tests
+    apt install -y rsync iproute2 krb5-kdc krb5-admin-server fakeroot lsof sudo python3.11 python3.11-dev python3-pip openjdk-11-jdk libipc-run-perl;\
+    pip install --upgrade pip; \
+# Install allure-behave for behave tests
+    pip3 install pytest gsutil behave~=1.2.6 coverage~=4.5 'mock<=5.0.0' allure-behave==2.4.0 psutil; \
+    pip3 cache purge
+
+# setup ssh configuration
+RUN set -eux; \
+    # ssh-keygen -f /etc/ssh/ssh_host_rsa_key -N '' -t rsa; \
+    ssh-keygen -f /etc/ssh/ssh_host_dsa_key -N '' -t dsa; \
+    sed -i -e 's|Defaults    requiretty|#Defaults    requiretty|' /etc/sudoers; \
+    sed -ri 's/UsePAM yes/UsePAM no/g;s/PasswordAuthentication yes/PasswordAuthentication no/g' /etc/ssh/sshd_config; \
+    sed -ri 's@^HostKey /etc/ssh/ssh_host_ecdsa_key$@#&@;s@^HostKey /etc/ssh/ssh_host_ed25519_key$@#&@' /etc/ssh/sshd_config
+
+# run environment for gpdb
+RUN set -eux; \
+    echo '#!/bin/sh' >> /etc/profile.d/jdk_home.sh; \
+    echo 'export JAVA_HOME=/etc/alternatives/java_sdk' >> /etc/profile.d/jdk_home.sh; \
+    echo 'export PATH=$JAVA_HOME/bin:$PATH' >> /etc/profile.d/jdk_home.sh
```

Comment:
Do we really need this in this image? It's not in the [commit](https://github.com/arenadata/gpdb/commit/c5d46dd4bb9a0601da22e3c89e9fb09e01414d74).

Reply:
It's in Dockerfile for 7.x for rockylinux. Any reasons to not have it for ubuntu?

Reply:
same as for 6X